library/src/main/java/com/pokegoapi/util/Signature.java

```diff
@@ -39,19 +39,6 @@ public static void setSignature(PokemonGo api, RequestEnvelopeOuterClass.Request
 				.setUnknown22(ByteString.copyFrom(uk22))
 				.setTimestamp(api.currentTimeMillis())
 				.setTimestampSinceStart(curTime - api.startTime)
```

Comment:
The ";" from the the end of setActivityStatus() needs to be at the end of this one.